tests/test-policies/src/test/java/org/mule/test/policy/PolicyTestCase.java

```diff
@@ -35,7 +35,8 @@
 
 import org.junit.Test;
 
-public class PolicyTestCase extends MuleArtifactFunctionalTestCase {
+//TODO MULE-12760 Make this test extend MuleArtifactFunctionalTestCase
+public class PolicyTestCase extends FunctionalTestCase {
```

Comment:
There seems to be some issue with the runner for this case of having tests in the same module as the plugin